src/index.html

```diff
@@ -67,6 +66,274 @@ <h4 id="title">I'm a software Developer</h4>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+   
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+  <div class="projects-done">
+    <div class="project-name">
+      <p class="name"> Professional Art</p>
+      <p class="name-data">Printing Data</p>
+    </div>  
+    <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+    <div class="project-stack">
+        <ul class="project-tools">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+        </ul>
+    </div>
+    <button class="see-project">See Project</button>
+  </div>
+
+
+
+  
```

Comment:
- [ ] Great job 💪on your efforts! You have done a lot of work👏. Kindly remove extra white spaces in your code. This is the best practice✔ and will help make your code cleaner and more organized.